superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts

```diff
@@ -64,7 +64,7 @@ describe('AdhocFilters', () => {
     });
   });
 
-  it('Set custom adhoc filter', () => {
+  xit('Set custom adhoc filter', () => {
```

Comment:
This PR just got a lot more controversial! I'm inclined to approve, but hope we can fix this test one day, as it HAS caught real errors for me before.

Reply:
I'M GOING CRAZY DEALING WITH FLAKY TESTS.

FYI that's the second flaky test in this module. Another one got fixed by @nytai yesterday